san/batch.py

```diff
@@ -23,8 +22,8 @@ def execute(self):
 
         idxs = sorted([int(k.split('_')[1]) for k in res.keys()])
         for idx in idxs:
-            key = "query_{}".format(idx)
-            df = convert_to_datetime_idx_df(res[key])
+            query = self.queries[idx][0].split("/")[0]
+            df = transform_query_result(idx, query, res)
```

Comment:
can you somehow hardcode the expected result? if the transformation is not applied when it has to this test will still pass as transformations won't be applied on both the result and the expected result

Reply:
Just finished up hardcoding the tests expected results